crates/alkahest/src/gui/outliner.rs

```diff
@@ -59,11 +62,43 @@ impl GuiView for OutlinerPanel {
 
             let camera = resources.get::<Camera>();
 
+            fn search(entity: Entity, s: &str, sce: &Scene) -> bool {
+                let e = sce.entity(entity);
+                let label = if let Some(label) = e.get::<Label>() {
+                    format!("{label}")
+                } else {
+                    return false;
+                };
```

Comment:
Unnecessary format
Can be written more idiomatically as such
```suggestion
                let Some(label) = e.get::<Label>().map(Label::to_string) else {
                    return false;
                };
```